docs/lib/graphqlapi/fragments/flutter/authz/20_oidc.md

```diff
@@ -0,0 +1,5 @@
+<amplify-callout>
+
+OIDC modes is not available just yet for Flutter. We are actively working on this. 
```

Comment:
are there multiple OIDC modes? We should either say `OIDC modes are ...` or `OIDC mode is ...`